docs/application/tizen-studio/setup/openjdk.md

```diff
@@ -70,16 +75,14 @@ This section explains how to install the OpenJDK version 12 for Ubuntu:
     ```
     sudo update-alternatives --config java
     ```
-    > **Note**
-    >
+    > [!NOTE]
     > If there are multiple JDK installed, select the required java version by entering the selection number.
```

Comment:
If there are multiple versions of JDK installed, select the required Java version by entering the selection number

Reply:
Please fix other comments as well